### DISCUSSION TITLE
i wanna refund my github action pro plan(yearly) and re-plan monthly

### DISCUSSION THREAD

**Discussion by yevgnenll** [community/community#108116] [New to GitHub]
Hi i subscribe pro start yesterday

i paid yearly but i wanna switch to monthly paid
could you refund my plan and re-paid?? or just refund??

i think dollar currency monthly good for me

**Comment by mw-xyz** [comment]
You should follow the steps [here](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/downgrading-your-accounts-plan). Let me know if this helps

**Reply by yevgnenll** [reply]
hi i alreay read before ask
but downgrade my account next billing date right?

i wanna refund and re-pay as monthly T^T
